src/torrent_peer.cpp

```diff
@@ -82,7 +82,7 @@ namespace libtorrent
 		if (e1.address() == e2.address())
 		{
 			if (e1.port() > e2.port())
-				swap(e1, e2);
```

Comment:
the convention when using swap() is actually to not fully qualify it, to allow argument dependent lookup. You may notice that there's a `using std::swap` above, which pulls in the standard overload if there is no better one.